src/shared_gui_components/BuildingComponentDialogCentralWidget.cpp

```diff
@@ -210,11 +238,11 @@ void BuildingComponentDialogCentralWidget::setTid(const std::string& filterType,
   m_collapsibleComponentList->setText(title);
 
   // the total number of results
-  int lastTotalResults = remoteBCL.lastTotalResults();
+  int lastTotalResults = m_allResponses.size();
   m_collapsibleComponentList->setNumResults(lastTotalResults);
 
   // the number of pages of results
-  int numResultPages = remoteBCL.numResultPages();
+  int numResultPages = (lastTotalResults + itemsPerPage - 1) / itemsPerPage;
```

Comment:
Is this the same as `(lastTotalResults / itemsPerPage) + 1`?

Reply:
Also make a comment when using integer division

Reply:
Yes, your suggestion is more straight forward though so I will use it